File: examples/test_backdoor.py
# Test Python code with suspicious constructs
import urllib.request
import base64

def malicious_function():
    # Suspicious eval
    user_input = input("Enter some code: ")
    eval(user_input)

    # Suspicious exec
    exec("print('Malicious code executed')")

    # Network call
    urllib.request.urlopen("http://malicious-site.example.com")

    # Obfuscated code
    obfuscated = base64.b64decode("cHJpbnQoIkhlbGxvIik=").decode()
    exec(obfuscated)

File: examples/test_backdoor.php
<?php
// Test PHP code with suspicious constructs

// Suspicious system
system("whoami");

// Suspicious shell_exec
$output = shell_exec("ls -la");

// Suspicious eval
eval('echo "Malicious PHP code";');

// Obfuscated code
$obfuscated = base64_decode("ZWNobyAiSGVsbG8iOw==");
eval($obfuscated);
?>